config/secrets.yml

```diff
@@ -12,10 +12,10 @@
 
 development:
   secret_key_base: 46b695793bbe16e27ec8ba5570143639d925a1fbaa5d792dea276c3e17b94bfb7f39be3640dea1203aa35a64aba7b41a61a380aa8cbe04fe7ba4204eb2a5511c
-  client_id: <%= ENV['WECROWD_CLIENT_ID'] %>
-  client_secret: <%= ENV['WECROWD_CLIENT_SECRET'] %>
-  account_id: <%= ENV['WECROWD_ACCOUNT_ID'] %>
-  access_token: <%= ENV['WECROWD_ACCESS_TOKEN'] %>
+  client_id: 53591
+  client_secret: 7fdb391773
+  account_id: 341673034
+  access_token: STAGE_58b9b287de2763ea30b4a1bf6a9893efeaf6195df66821ffbb23924756c52afa
```

Comment:
You should also revoke these credentials, since they’ve been made public.


On June 22, 2015 at 4:15:47 PM, Facundo Ramos (notifications@github.com) wrote:

In config/secrets.yml:

> @@ -12,10 +12,10 @@
>   
>  development:
>    secret_key_base: 46b695793bbe16e27ec8ba5570143639d925a1fbaa5d792dea276c3e17b94bfb7f39be3640dea1203aa35a64aba7b41a61a380aa8cbe04fe7ba4204eb2a5511c
> -  client_id: <%= ENV['WECROWD_CLIENT_ID'] %>
> -  client_secret: <%= ENV['WECROWD_CLIENT_SECRET'] %>
> -  account_id: <%= ENV['WECROWD_ACCOUNT_ID'] %>
> -  access_token: <%= ENV['WECROWD_ACCESS_TOKEN'] %>
> +  client_id: 53591
> +  client_secret: 7fdb391773
> +  account_id: 341673034
> +  access_token: STAGE_58b9b287de2763ea30b4a1bf6a9893efeaf6195df66821ffbb23924756c52afa
You should remove this from the pull request.

—
Reply to this email directly or view it on GitHub.